server/backend/quiz/views.py

```
@@ -0,0 +1,108 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework.permissions import IsAuthenticated
+from .models import Category, Question, Option, QuizAttempt, UserAnswer
+from .serializers import CategorySerializer, QuestionSerializer
+
+class CategoryViewSet(viewsets.ReadOnlyModelViewSet):
+    queryset = Category.objects.all()
+    serializer_class = CategorySerializer
+
+class QuizAPIView(APIView):
+    def get(self, request, category_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `get`

(ANN201)